README.md

```diff
@@ -192,6 +192,40 @@ import * as stats from './stats';
 export { fallback } from './fallback';
 ```
 
+## How to add allowed word
+
+All the allowed words are located in an array in ` src/config.ts ` inside the object ` ALEX ` as shown below. 
+
+```ts
+ALEX: {
+    profanitySureness: 2,
+    noBinary: true,
+    allow: [
+      'just',
+      'brother-sister',
+      'brothers-sisters',
+      'daft',
+      'master',
+      'clearly',
+      'moaning',
+      'host-hostess',
+      'husband-wife',
+      'obvious',
+      'simple',
+      'of-course',
+      'special',
+      'dad-mom',
```

Comment:
I don't think we should have the whole object here.
maybe like this:
```ts
ALEX: {
    profanitySureness: 2,
    noBinary: true,
    allow: [
      'just',
      'brother-sister',
      'brothers-sisters',
      'daft',
      ...
```
I don't want to add more words to the readme as the config gets larger.

What do you think?

Reply:
It is a great idea.